src/core/stdc/assert_.d

```diff
@@ -0,0 +1,52 @@
+/**
+ * D header file for C99.
+ *
+ * $(C_HEADER_DESCRIPTION pubs.opengroup.org/onlinepubs/009695399/basedefs/_assert.h.html, _assert.h)
+ *
+ * License: Distributed under the
+ *      $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0).
+ *    (See accompanying file LICENSE)
+ * Source:    $(DRUNTIMESRC core/stdc/_assert_.d)
+ * Standards: ISO/IEC 9899:1999 (E)
+ */
+
+module core.stdc.assert_;
+
+extern (C):
+@trusted:
+nothrow:
+@nogc:
```

Comment:
I agree with @ZombineDev, should be `pure`. The betterC purists will come back at you anyway without it.